client/dashboard/utils/site-features.ts

```diff
@@ -11,3 +11,14 @@ export const canUpdateWordPressVersion = ( site: Site ): boolean => site.is_wpco
 export const canSetStaticFile404Handling = ( site: Site ): boolean => site.is_wpcom_atomic;
 
 export const canGetPrimaryDataCenter = ( site: Site ): boolean => site.is_wpcom_atomic;
+
+export const canUpdateLegacyFeatures = ( site: Site ): boolean => {
+	if ( ! site.plan ) {
+		return false;
+	}
+
+	const { plan } = site;
+	return [ 'locked-mode', 'legacy-contact' ].some( ( feature ) =>
```

Comment:
Maybe worth adding them here 

https://github.com/Automattic/wp-calypso/blob/f5a070cc0a75b07935d241495ef81e61fbd8b6ba/client/dashboard/data/constants.ts#L1-L4

Reply:
Somehow I missed this. Yes, it should be added here!